spec/readwise_spec.rb

```diff
@@ -1,3 +1,5 @@
+require "readwise/version"
+
```

Comment:
Spec was failing without this require statement

Reply:
That's strange, even with `bundle exec`?

Reply:
Ah, it does run with bundle exec rake spec, I'll update the README to indicate that being the case.